libbeat/_meta/config/output-elasticsearch.reference.yml.tmpl

```diff
@@ -81,6 +81,14 @@ output.elasticsearch:
   #allow_older_versions: false
 
 {{include "ssl.reference.yml.tmpl" . | indent 2 }}
+
+  # Enable exiting {{.BeatName}} if any of the files pointed by
+  # `ssl.certificate_authorities` changes
+  #ssl.exit_on_cert_change.enabled
+
+  # Interval to scan for changes on CA certificate files
+  #ssl.exit_on_cert_change.period
```

Comment:
Now the real hard question: Is the `ssl` a good place for this config? I thought about putting on the output config or even as a general beat config parameter, but I settled with `ssl` because it's specific for those certs.